grid2op/tests/test_Action.py

```diff
@@ -772,6 +813,13 @@ def test_extract_from_vect(self):
         res = self.helper_action.extract_from_vect(vect, "_set_line_status")
         assert np.all(res == act._set_line_status)
 
+    def test_sample(self):
+        try:
```

Comment:
to avoid non reproducible behaviour, please seed the action space when testing the sampling (i would have to have tests that randomly passes or fails)